RecordCollection/Models/AlbumViewModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace RecordCollection.Models
+{
+    public class AlbumViewModel
+    {
+        public Album Album { get; set; }
+        public List<Album> RelatedAlbums { get; set; }
+        public int RelatedAlbumsCount { get; set; }
+
+        public bool IsModernAlbum()
```

Comment:
I think we can move this method. `IsModernAlbum()` would be better suited to the Album model. We may need to make this determination again outside of an album view.